src/hacbs/components/PipelineRunDetailsView/PipelineRunDetailsSection.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { SectionHeading } from '../../shared/utils/headings';
+import { PipelineRunKind } from '../../types';
+
+export interface PipelineRunDetailsSectionProps {
+  pipelineRun: PipelineRunKind;
+}
+
+const PipelineRunDetailsSection: React.FC<PipelineRunDetailsSectionProps> = ({ pipelineRun }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <SectionHeading text={t('pipelines-plugin~PipelineRun details')} />
+      <div className="row">
+        <div className="col-sm-6">{pipelineRun.metadata.name}</div>
+        <div className="col-sm-6">{/* <ResourceSummary resource={pipelineRun} /> */}</div>
```

Comment:
Use patternfly css or components instead of boostrap

Reply:
i didn't wind up keeping this file